src/interpreter/context.rs

```diff
@@ -173,7 +173,7 @@ impl Context {
             *new_start += Duration::try_minutes(minutes).unwrap();
             *new_end += Duration::try_minutes(minutes).unwrap();
             log::debug!(
-                "Set time frame to ({}, {}) (seeked to {} minutes backward)",
+                "Set time frame to ({}, {}) (sought to {} minutes backward)",
```

Comment:
So as this

Reply:
This is, but sounds odd/old fashion - https://github.com/azat/chdig/commit/23b37061f447d828f224f6b9da7fbe94f58979c3